Makefile

```diff
@@ -1,12 +1,38 @@
-CUDAPATH=/usr/local/cuda
+CUDAPATH ?= /usr/local/cuda
 
-# Have this point to an old enough gcc (for nvcc)
-GCCPATH=/usr
+NVCC     :=  ${CUDAPATH}/bin/nvcc
+CCPATH   ?=
```

Comment:
Made this by default blank as most systems have GCC in the default path. This can be easily overriden while invoking make for a non standard install.